docs/developer-docs/latest/plugins/graphql.md

```diff
@@ -341,33 +339,207 @@ The object describing the extension accepts the following parameters:
 | `typeDefs`        | String | Allows extending the schema types using [GraphQL SDL](https://graphql.org/learn/schema/)     |
 | `plugins`         | Array  | Allows extending the schema using Nexus [plugins](https://nexusjs.org/docs/plugins)          |
 | `resolvers`       | Object | Defines custom resolvers                                                                     |
-| `resolversConfig` | Object | Defines configuration options for the resolvers                                              |
+| `resolversConfig` | Object | Defines [configuration options for the resolvers](#custom-configuration-for-resolvers), such as [authorization](#authorization-configuration), [policies](#policies) and [middlewares](#middlewares) |
 
 ::::tip
 The `types` and `plugins` parameters are based on [Nexus](https://nexusjs.org/). To use them, register the extension as a function that takes `nexus` as a parameter:
 
 ::: details Example:
+
 ```js
-// path: /config/functions/register.js
-
-const extension = ({ nexus }) => ({
-  types: [
-    nexus.objectType({
-      …
-    }),
-  ],
-  plugins: [
-    nexus.plugin({
-      …
+
+// path: ./src/index.js
+
+module.exports = {
+  register({ strapi }) {
+    const extension = ({ nexus }) => ({
+      types: [
+        nexus.objectType({
+          …
+        }),
+      ],
+      plugins: [
+        nexus.plugin({
+          …
+        })
+      ]
     })
-  ]
-})
 
-strapi.plugin('graphql').service('extension').use(extension)
+    strapi.plugin('graphql').service('extension').use(extension)
+  }
+}
 ```
+
 :::
 ::::
 
+#### Custom configuration for resolvers
+
+A resolver is a GraphQL query handler (i.e. a function, or a collection of functions, that generate(s) a response for a GraphQL query). Each field has a default resolver.
+
+When [extending the GraphQL schema](#extending-the-schema), the `resolversConfig` key can be used to define a custom configuration for a resolver, which can include:
+
+* [authorization configuration](#authorization-configuration) with the `auth` key
+* [policies with the `policies`](#policies) key
+* and [middlewares with the `middlewares`](#middlewares) key
+
+##### Authorization configuration
+
+<!-- TODO: consider reworking the TOC as having h5s is not optimal… -->
+By default, the authorization of a GraphQL request is handled through the [Users & Permissions plugin](#usage-with-the-users-permissions-plugin). A request is allowed if the appropriate permissions are given: for instance, if a 'Category' content-type exists and is queried through GraphQL with the `Query.categories` handler, the request is allowed if the appropriate `find` and/or `findOne` permission(s) for the 'Categories' content-type are given.
+
+To change how this authorization is configured, use the `resolversConfig.auth` key that accepts:
```

Comment:
It's not really `resolversConfig.auth` though. We're using the auth attribute of a resolver config defined within resolversConfig
eg: `resolversConfig: { MyResolverName: { auth: {} } }`
Do you have another suggestion for this sentence ?